packages/wallet-sdk/src/relay/aes256gcm.test.ts

```diff
@@ -47,15 +49,6 @@ describe("aes256gcm", () => {
       "secret must be 256 bits",
     );
 
-    await expect(
-      decrypt("text", secret).subscribe(
-        () => {
-          fail("expected error");
-        },
-        error => {
-          expect(error).toBe("expected error");
```

Comment:
not sure how this worked before as well.
`error`'s message is not equal to `expected error`

Reply:
cc @erin-at-work